### PR TITLE
[Tests-Only] refactor files and rename tests to not use skeleton files

### PR DESCRIPTION
**Pull request by dpakach**
## Description

refactor files and rename tests to not use skeleton files

## Related Issue

- Related https://github.com/owncloud/QA/issues/659 

## Motivation and Context

Make CI faster

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIOCISFiles3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14630/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14630/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14630/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIOCISFiles3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14695/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14695/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14695/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14695/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIOCISFiles3</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14702/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Visual regression tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14702/

<p>


Diff Image: </br>

Actual Image: </br>

Comparing Against: </br>


</p></details>
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14702/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14702/

<p>





</p></details>

File: tests/acceptance/features/webUIFilesSearch/search.feature [hunk 1]
```diff
@@ -6,8 +6,18 @@ Feature: Search
   So that I can find needed files quickly
 
   Background:
-    Given user "Alice" has been created with default attributes
-    And user "user0" has been created with default attributes
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And user "user0" has been created with default attributes and without skeleton files
```
Comment (swoichha) [inline tests/acceptance/features/webUIFilesSearch/search.feature:10]:
```suggestion
    Given these users have been created with default attributes and without skeleton files:
      | username |
      | Alice    |
      | user0    |
```

File: tests/acceptance/features/webUIFilesSearch/search.feature [hunk 2]
```diff
@@ -6,8 +6,18 @@ Feature: Search
...
+    And user "Alice" has created the following folders
+      | entry_name     |
+      | simple-folder    |
+      | simple-empty-folder |
+      | strängé नेपाली folder   |
```
Comment (swoichha) [inline tests/acceptance/features/webUIFilesSearch/search.feature:15]: Indentation can be fixed here.

File: tests/acceptance/features/webUIRenameFiles/renameFilesInsideProblematicFolderName.feature [hunk 3]
```diff
@@ -4,28 +4,33 @@ Feature: Renaming files inside a folder with problematic name
...
+    And user "Alice" has logged in using the webUI
+    When the user opens folder "<folder>" using the webUI
     And the user renames file "lorem.txt" to "???.txt" using the webUI
     Then file "???.txt" should be listed on the webUI
     When the user reloads the current page of the webUI
     Then file "???.txt" should be listed on the webUI
     Examples:
       | folder                  |
-      | "strängé नेपाली folder" |
+      | strängé नेपाली folder |
```
Comment (swoichha) [inline tests/acceptance/features/webUIRenameFiles/renameFilesInsideProblematicFolderName.feature:36]: fix indentation

File: tests/acceptance/features/webUIRenameFiles/renameFilesInsideProblematicFolderName.feature [hunk 4]
```diff
@@ -4,28 +4,33 @@ Feature: Renaming files inside a folder with problematic name
...
     And the user renames file "lorem.txt" to "???.txt" using the webUI
     Then file "???.txt" should be listed on the webUI
     When the user reloads the current page of the webUI
     Then file "???.txt" should be listed on the webUI
     Examples:
       | folder           |
-      | "0"              |
-      | "'single'quotes" |
+      | 0              |
+      | 'single'quotes |
```
Comment (swoichha) [inline tests/acceptance/features/webUIRenameFiles/renameFilesInsideProblematicFolderName.feature:22]: fix indentation

File: tests/acceptance/features/webUIFilesSearch/search.feature [hunk 5]
```diff
@@ -6,8 +6,20 @@ Feature: Search
...
-    And user "user0" has been created with default attributes
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | user0    |
+    And user "Alice" has created the following folders
+      | entry_name            |
+      | simple-folder         |
+      | simple-empty-folder   |
+      | strängé नेपाली folder |
```
Comment (swoichha) [inline tests/acceptance/features/webUIFilesSearch/search.feature:17]: fix indentation

**Review by Talank — COMMENTED** [review]
LGTM other than already existing reviews.

**Review by individual-it — APPROVED** [review]

**Pull request merged**